assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -501,28 +511,44 @@ che.infra.kubernetes.trusted_ca.dest_configmap_labels=
 
 ### OpenShift Infra parameters
 
-# Since OpenShift infrastructure reuse Kubernetes infrastructure components
-# OpenShift infrastructure reuse most of the Kubernetes configuration attributes.
+# Since OpenShift infrastructure reuses Kubernetes infrastructure components
+# OpenShift infrastructure reuses most of the Kubernetes configuration attributes.
 
-# DEPRECATED - please do not change the value of this property otherwise the existing workspaces will loose data. Do not
+# DEPRECATED - please do not change the value of this property otherwise the existing workspaces will lose data. Do not
 # set it on new installations.
 #
 # Defines OpenShift namespace in which all workspaces will be created.
 # If not set, every workspace will be created in a new project, where project name = workspace id
 # It's possible to use <username> and <userid> placeholders (e.g.: che-workspace-<username>).
-# In that case, new project will be created for each user. OpenShift oauth or service account with
+# In that case, new project will be created for each user. OpenShift OAuth or service account with
 # permission to create new projects must be used.
 #
 # If the project pointed to by this property exists, it will be used for all workspaces. If it does not exist,
 # the namespace specified by the che.infra.kubernetes.namespace.default will be created and used.
 che.infra.openshift.project=
 
+<<<<<<< HEAD
 # Comma separated list of labels to add to the CA certificates config map in user workspace.
 # See che.infra.kubernetes.trusted_ca.dest_configmap property.
 # This default value is used for automatic cluster CA bundle injection in Openshift 4.
 che.infra.openshift.trusted_ca.dest_configmap_labels=config.openshift.io/inject-trusted-cabundle=true
+=======
+
+# Configures name of the trust-store config map where the CA bundles are stored in Openshift 4.
+# This map is supposed to be initially created by Che installer (operator, etc) with basically
+# any name, and Che server finds it by specific label (see below) during workspace startup and then
+# creates and mounts the same map in the namespace of the workspace. The property defines the name of the
+# map in the workspace namespace.
+che.infra.openshift.trusted_ca_bundles_config_map=ca-certs
+
+# Label name for config maps which are used for automatic certificate injection in Openshift 4.
+che.infra.openshift.trusted_ca_bundles_config_map_labels=config.openshift.io/inject-trusted-cabundle=true
+
+# Configures path on workspace containers where the CA bundles are mount.
+che.infra.openshift.trusted_ca_bundles_mount_path=/public-certs
+>>>>>>> 9e06a3d88e... Update language.
```

Comment:
looks like wrong merge conflict resolution. please fix or it will probably make the property file invalid